src/kv/kv_types.h

```diff
@@ -372,17 +372,14 @@ namespace kv
   struct PendingTxInfo
   {
     CommitResult success;
-    TxHistory::RequestID reqid;
```

Comment:
This was unused, so I've removed it. There's still a `reqid` on each `Tx`, which we put into the `AFT_REQUESTS` table, but we didn't use it when we got it out of this `PendingTxInfo`.